src/ts/props/button.ts

```diff
@@ -0,0 +1,33 @@
+import { MantineRadius, MantineSize } from "@mantine/core";
+import React from "react";
+
+export interface __CloseButtonProps {
+    /** Controls width and height of the button. Numbers are converted to rem. `'md'` by default. */
+    size?: MantineSize | (string & {}) | number;
```

Comment:
`size` looks like the only difference between these two interfaces - if that's real, perhaps make them both inherit from a `__BaseButtonProps` or some such, to reduce duplication?